### ISSUE TITLE
'Invalid authentication token' when rendering images sent in a chainlit application with authentication enabled.

### ISSUE THREAD

**Issue by thiagotps** [Chainlit/chainlit#1485]
**Describe the bug**
When there is no authentication, images sent in the chat are rendered normally. But with authentication set up, images sent over the chat are not rendered by the browser, receiving a 401 error with message 
```json
{"detail":"Invalid authentication token"}
```
when accessed.  

**To Reproduce**
Just try to send an image to the chat of the chainlit app executed through the following command
```bash
chainlit run test_image.py -w
```
with the following code
```python
#!/usr/bin/env python3
import chainlit as cl


@cl.password_auth_callback
def auth_callback(username: str, password: str):
    # Fetch the user matching username from your database
    # and compare the hashed password with the value stored in the database
    if (username, password) == ("admin", "admin"):
        return cl.User(
            identifier="admin", metadata={"role": "admin", "provider": "credentials"}
        )
    else:
        return None


@cl.on_chat_start
async def on_chat_start():
    if app_user := cl.user_session.get("user"):
        await cl.Message(f"Hello {app_user.identifier}").send()


@cl.on_message
async def on_message(message: cl.Message):
    await cl.Message(f"Message Received:{message.content}").send()
```

**Expected behavior**
I expect to see the image being rendered normally. 

**Screenshots**
![image](https://github.com/user-attachments/assets/76ddbbeb-fe3a-4e8a-84ef-5efa37412b28)
And when trying to directly open the image with the http://localhost:8000/project/file/e9901e54-7210-468b-8ebb-218426bbf5f9?session_id=a9d63b80-0432-4e98-8fb6-5bde5a83d11b link:
![image](https://github.com/user-attachments/assets/c05cb887-6f41-4541-b5d9-597bdf566e91)

**Desktop (please complete the following information):**

- OS: Arch Linux
- Browser: Firefox 
- Version: 131.0.3

**Additional context**
chainlit version is 1.3.0

**Comment by gloveboxes** [comment]
This is a bug introduced with Chainlit 1.3. It's still there in Chainlit 1.3.1. 

In reqirements.txt set chainlit==1.2.0 and you'll be good to go till resolved.

Cheers Dave

**Comment by dokterbob** [comment]
@gloveboxes @thiagotps Are you 100% sure you are, in fact, running 1.3.1? 

On first sight, this seems a lot like exactly the bug (#1472) we attempted to fix with the release 1.3.1.

**Comment by thiagotps** [comment]
> @gloveboxes @thiagotps Are you 100% sure you are, in fact, running 1.3.1?
> 
> On first sight, this seems a lot like exactly the bug (#1472) we attempted to fix with the release 1.3.1.

I was running version 1.3.0. With version 1.3.1 it is indeed working.

**Comment by dokterbob** [comment]
@thiagotps Thanks for the head's up!
